From e21281e1a836ac25a588168cfa875018c5b52b62 Mon Sep 17 00:00:00 2001
From: Anthony Lazara <me@a5y.rocks>
Date: Mon, 5 Oct 2026 22:06:51 +0800
Subject: [PATCH] about: rm donate row from about page

GNOME is good stuff, but they also have a rotting community that is self-righteous. Really unfortunate.
---
 panels/system/about/cc-about-page.blp         | 19 -------------------
 panels/system/about/cc-about-page.c           |  7 -------
 .../org.gnome.Settings.metainfo.xml.in.in     |  1 -
 3 files changed, 27 deletions(-)

diff --git a/panels/system/about/cc-about-page.blp b/panels/system/about/cc-about-page.blp
index 47e8fa971..a46180d26 100644
--- a/panels/system/about/cc-about-page.blp
+++ b/panels/system/about/cc-about-page.blp
@@ -25,25 +25,6 @@ template $CcAboutPage: Adw.NavigationPage {
           }
         }
 
-        Adw.PreferencesGroup {
-          Adw.ActionRow {
-            title: _("Support GNOME");
-            subtitle: _("GNOME needs your help. Please donate today.");
-
-            [suffix]
-            Button {
-              label: _("D_onate");
-              use-underline: true;
-              valign: center;
-              styles [
-                "suggested-action",
-              ]
-
-              clicked => $on_donate_button_clicked_cb(template);
-            }
-          }
-        }
-
         Adw.PreferencesGroup {
           $CcHostnameEntry hostname_row {
             title: _("_Device Name");
diff --git a/panels/system/about/cc-about-page.c b/panels/system/about/cc-about-page.c
index 70d12272f..536cc334a 100644
--- a/panels/system/about/cc-about-page.c
+++ b/panels/system/about/cc-about-page.c
@@ -132,12 +132,6 @@ setup_os_logo (CcAboutPage *self)
 #endif
 }
 
-static void
-on_donate_button_clicked_cb (CcAboutPage *self)
-{
-    g_app_info_launch_default_for_uri ("https://donate.gnome.org/", NULL, NULL);
-}
-
 static void
 on_release_notes_button_clicked_cb (CcAboutPage *self)
 {
@@ -1031,7 +1025,6 @@ cc_about_page_class_init (CcAboutPageClass *klass)
     gtk_widget_class_bind_template_child (widget_class, CcAboutPage, virtualization_row);
 
     gtk_widget_class_bind_template_callback (widget_class, on_copy_row_activated_cb);
-    gtk_widget_class_bind_template_callback (widget_class, on_donate_button_clicked_cb);
     gtk_widget_class_bind_template_callback (widget_class, on_release_notes_button_clicked_cb);
 }
 
diff --git a/shell/appdata/org.gnome.Settings.metainfo.xml.in.in b/shell/appdata/org.gnome.Settings.metainfo.xml.in.in
index 20eac8a65..9b13eabce 100644
--- a/shell/appdata/org.gnome.Settings.metainfo.xml.in.in
+++ b/shell/appdata/org.gnome.Settings.metainfo.xml.in.in
@@ -29,7 +29,6 @@
   <url type="homepage">https://apps.gnome.org/Settings/</url>
   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-control-center/issues/</url>
   <url type="contact">https://discourse.gnome.org/tag/settings</url>
-  <url type="donation">https://www.gnome.org/donate/</url>
   <url type="help">https://discourse.gnome.org/tag/settings</url>
   <url type="translate">https://l10n.gnome.org/module/gnome-control-center/</url>
   <url type="contribute">https://welcome.gnome.org/app/Settings/</url>
-- 
2.55.0

